style.css

```diff
@@ -590,3 +557,106 @@ li {
   background: #1c1a19;
   justify-content: space-around;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  padding: 80px 24px 0 24px;
+  gap: 10px;
+}
+
+.contact-form p {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: center;
+  gap: 24px;
+  height: 300px;
+}
+
+.contact-form img {
+  margin-top: 50px;
+}
+
+fieldset {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: none;
+  gap: 20px;
+  order: 2;
+}
+
+textarea {
+  font-style: italic;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  background-image: url(Images/Container.png);
+  width: 306px;
+  height: 114px;
+  border: none;
+  margin-top: 10px;
+}
+
+textarea:hover {
+  border: 1px solid #ff6b00;
+  background-image: url(Images/Icon\ -\ Resizer\ Handle.png);
+  background-repeat: no-repeat;
+  mix-blend-mode: normal;
+}
+
+input {
+  border-bottom: 1px solid #dbd8d7;
+  width: 306px;
+  height: 48px;
+  border: none;
+  padding-left: 10px;
+}
+
+input.button {
+  background: #ff6b00;
+  padding: 12px;
+  width: 131px;
+  color: #fff;
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  font-weight: 700;
+  order: 3;
+  margin-top: 15px;
+}
+
+input.button:hover {
+  background-color: #ff6b00;
+  border: 1px solid white;
+}
+
+input.button:active {
+  background-color: #ff6b00;
+  border: 1px solid white;
+}
```

Comment:
- [OPTIONAL] Well done implementing the `hover` and `active` interactions 🥇 However, these are not according to the button interactions given in Figma template.  I highly recommend implementing it 😄✨

### _Your button interactions 🔽_
Hover | Active
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/228839543-45b560bd-3d44-4b88-8936-785baad84f80.png) | ![image](https://user-images.githubusercontent.com/106119105/228839630-b8e70836-0507-430a-b378-9c8af60f3f89.png) |

### _Figma button interactions🔽_
![image](https://user-images.githubusercontent.com/106119105/228838846-997d73c2-a4aa-44b8-9bc2-e13971f00063.png)